pyproject.toml

```diff
@@ -48,6 +48,10 @@ types-toml = "*"
 types-redis = "*"
 pytest-redis = "^2.4.0"
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
Probably should merge the above `tool.poetry.dev-dependencies` into this section?

Reply:
Oh yeah I guess my poetry version is too new. We will have to check this for poetry backwards compatibility and leave a note in the docs if its not backwards-compatible.